### ISSUE TITLE
🐛 [bug] [easy] v18 breaks instant send

### ISSUE THREAD

**Issue by coolaj86** [dashpay/insight-ui#75]
I'm getting this from https://insight.dash.org when trying to use Instant Send:

> Second argument must be numeric (maxfeerate) and no longer supports a boolean. To allow a transaction with high fees, set maxfeerate to 0.. Code:-8

The request looks like this:

```js
let reqObj = {
  method: "POST",
  url: `${baseUrl}/insight-api-dash/tx/sendix`,
  form: {
    rawtx: hexTx,
  },
};
```

Since I'm sending a hex `tx` and the error message is about positional arguments to a function call, which makes me think that ***Dash Insight*** is out of sync with the latest deployed version of `dashd` or some such, on the Dash Insight server.

It stopped working Sunday, ostensibly when Dash Insight was upgraded to v18?

It sounds like there's somewhere that `maxfeerate` is set to `false` and instead needs to be set to `0`.

**Comment by coolaj86** [comment]
This issue needs to be transferred to https://github.com/dashevo/insight-api

**Comment by strophy** [comment]
Probably resolved in #81, please confirm

**Comment by riongull** [comment]
@coolaj86, please take a look at this.

**Comment by coolaj86** [comment]
The changes that I made I've tested and the API is up on dashsight.dashincubator.dev.

I haven't tested things that have changed from v0.19 to v0.20 that are up on `testnet` (https://testnet-insight.dashevo.org/insight/) because there are still some hardcoded places in my tools that use magic bytes for `mainnet`.

If this is up on `mainnet`, then I can test it. Otherwise I have to make updates first.

However, if the v0.19 to v0.20 changes are working for all other functionality, and the new deploy includes my changes that were included in the latter v0.19 (and I doubt they were removed), then at worst v0.20 won't break anything that wasn't already broken and at best it will work just as well as the v0.19 that we deployed on dashsight.dashincubator.dev but was never deployed to insight.dash.org did.

Alternatively I could try to deploy the v0.20 stuff on dashsight.dashincubator.dev, but that's a pain because `dashnode` has its own bespoke dependency management system outside of npm that doesn't work with git branch urls and such - requires installing, manually copying files over top of the installed files, etc - unless this is already published to npm in which case I can just update it.
